translations/bnpchs-2018-penyisihan-penjumlahan-banyak-angka.html

```diff
@@ -0,0 +1,63 @@
+<!--adding many numbers-->
+
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Jelix Fingga is a math lecturer at the Panda University. One day, Jelix Fingga gives his students numbers from $$$A$$$ to $$$B$$$ inclusive. He asks them to count the sum of the numbers from $$$A$$$ to $$$B$$$.
+</p>
+
+<p style="text-align:justify">
+  However, most of his students could not answer this question because the result can be very large. Seeing that, Jelix Fingga changed the question from counting the sum of these numbers to deciding whether the sum of the numbers from $$$A$$$ to $$$B$$$ is negative or positive.
```

Comment:
```suggestion
  However, most of his students could not answer this question because the result can be very large. Seeing that, Jelix Fingga changed the question from counting the sum of these numbers to deciding whether the sum of the numbers from $$$A$$$ to $$$B$$$ is negative, positive, or zero.
```